data/templates/accel-ppp/ppp-options.j2

```diff
@@ -0,0 +1,35 @@
+#ppp options
+[ppp]
+verbose=1
+check-ip=1
+ccp={{ "0" if ppp_options.ccp_disable is vyos_defined else "1" }}
```

Comment:
Can we do not use `ccp-disable`? 
If `ccp` is not configured, it is disabled. And we already have such behavior it for PPPoE
```
vyos@r4#  set service pppoe-server ppp-option 
Possible completions:
   ccp                  CCP negotiation (default disabled)
```

Reply:
It is enabled by default in accel-ppp. And it is enabled in other vendors by default.

Reply:
In the comment above, it says that `default disabled` in the current version
```
vyos@r4#  set service pppoe-server ppp-options 
Possible completions:
   ccp                  CCP negotiation (default disabled)

```

Reply:
ccp is enabled by default. I changed from ccp-disable to disable-ccp.